src/pages/K8s.vue

```diff
@@ -24,6 +24,15 @@
     </button>
     Resetting Kubernetes to default will delete all workloads and configuration
     <hr>
+    <system-preferences
+      :memory-in-g-b="settings.kubernetes.memoryInGB"
```

Comment:
That's… not great. Can we just call this `memory`, `processors`, `memory-avail`, `processors-avail` or something? What kind of names do other components usually get?

Reply:
At some point I think I read best to avoid single-word names in Vue because of possible conflicts with builtin tags. And I wanted the JS names to take precedence over the rotisserized HTML names because they're used in many places, and the tag names are only used at the point of call. I agree that it isn't great, but I wanted the units to be in the name, and that causes the stretched out attribute name.